internal/incoming/odfi/processor.go

```diff
@@ -89,6 +89,7 @@ func ProcessFiles(logger log.Logger, dl *downloadedFiles, alerters alerting.Aler
 
 	for _, processingPath := range []string{agent.InboundPath(), agent.ReconciliationPath(), agent.ReturnPath()} {
 		where := filepath.Join(dl.dir, processingPath)
+		validation.AllowMissingBatchHeader = processingPath == agent.ReconciliationPath()
```

Comment:
Can this be enabled by configuration?

Reply:
hmmm

Reply:
this one seems a bit harder to get a configuration available this low in the code.